samples/Net7Worker/EventHubCancellationToken.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace Net7Worker
+{
+    public class EventHubCancellationToken
+    {
+        private readonly ILogger _logger;
+
+        public EventHubCancellationToken(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<EventHubCancellationToken>();
+        }
+
+        [Function(nameof(EventHubCancellationToken))]
+        public void Run(
+            [EventHubTrigger("sample-workitems", Connection = "EventHubConnection")] string[] messages,
+            FunctionContext context,
+            CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("C# EventHub trigger function processing a request.");
+
+            try
+            {
+                foreach (var message in messages)
+                {
+                    cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Is the intent here to demonstrate a cancellation token invoked by the underlying SDK? If so, I'd have a hard time guessing that, given this is an isolated worker and the process doesn't work directly with the SDK. Perhaps the sample should have something in the readme.md to describe the possible scenarios where the cancellation token would be activated. Thank you.

Reply:
I'll make a note in the readme, but the public docs should be updated with this information (and new sample) @cloudmelon